packages/website/utils/contentfulSidebarLinks.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Should this file be gitignored then?

Reply:
I didn't ignore it because the other one was not ignored

Should I ignore both of them?

Reply:
I'd say so, it should be generated on build, right?

Reply:
![Screenshot 2022-04-25 at 15 13 26](https://user-images.githubusercontent.com/6597467/165096269-9eabce0a-a3b8-4a2c-b874-c10a8fdbc582.png)

we were wrong, we need these files